python/servo/package_commands.py

```diff
@@ -96,8 +96,11 @@ def copy_dependencies(binary_path, lib_path):
 
 def copy_windows_dependencies(binary_path, destination):
     deps = [
+        "api-ms-win-crt-runtime-l1-1-0.dll",
         "libcryptoMD.dll",
         "libsslMD.dll",
+        "msvcp140.dll",
+        "vcruntime140.dll",
```

Comment:
Great, thanks. That was my only concern, just to make sure we were good to go.

@aneeshusa - :+1: from me